### ISSUE TITLE
[TF2] Other weapons missing skins/styles.

### ISSUE THREAD

**Issue by Arbitz** [ValveSoftware/Source-1-Games#4293]
Certain other weapons have unused or missing alternate skins in the files that haven't been implemented ingame.

Thermal Thruster - Unused pink pyroland style texture (Found in C_Rocketpack_Fun)
Cozy Camper - Green skin from original workshop submission (Still on the backpack icon)
Holiday Punch - Brown "Officer's Ushanka" style from original workshop submission

**Comment by matty4kek** [comment]
I don't think It's a bug, It's just Valve chosing not to implement them and just pick an original, single style instead.
There's a lot of unused content in the game files.

**Comment by Arbitz** [comment]
> I don't think It's a bug, It's just Valve chosing not to implement them and just pick an original, single style instead. There's a lot of unused content in the game files. Meanwhile the missing Sydney Sleeper BLU skin was just suppose to be in the game, but it was just straight up missing.

This was moreso a feature request.

**Comment by TF2CutContentWiki** [comment]
> I don't think It's a bug, It's just Valve chosing not to implement them and just pick an original, single style instead. There's a lot of unused content in the game files. Meanwhile the missing Sydney Sleeper BLU skin was just suppose to be in the game, but it was just straight up missing.

True. The Pyromancer's mask model has an unused bodygroup submodel for the horns that changes how they look. This was never implemented ingame.

**Comment by condor00fr** [comment]
would be cool to see them anyway
